### PR TITLE
feat: Xamarin docs

### PR DESCRIPTION
**Pull request by lucas-zimerman**
The following pull request adds to sentry-docs the Xamarin and Xamarin forms doc page.

### CODE REVIEW

**Review by bruno-garcia — COMMENTED** [review]
You'll need a few more changes though:

https://sentry-docs-git-fork-lucas-zimerman-feat-xamarin-docs.sentry.dev/platforms/dotnet/guides/xamarin/ignoring-exceptions/

This snippet would need to be rewritten:
This file: https://github.com/getsentry/sentry-docs/blob/master/src/platforms/dotnet/common/ignoring-exceptions.mdx

Could be **copied** to `dotnet/guides/xamarin/common/ignoring-exceptions.mdx` and changed to `SentryXamarin.Init`.

File: src/includes/set-release/xamarin.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+```
+
+The SDK attempts to locate the release to add to events sent to Sentry.
+
+<Note>
+
+[Default values like 1.0 or 1.0.0.0 are ignored](https://github.com/getsentry/sentry-dotnet/blob/dbb5a3af054d0ca6f801de37fb7db3632ca2c65a/src/Sentry/Internal/ApplicationVersionLocator.cs#L14-L21).
+
+</Note>
+
```
Comment (bruno-garcia) [inline src/includes/set-release/xamarin.mdx:14]: Actually this is no longer true on 3.0

```suggestion
```

**Review by bruno-garcia — APPROVED** [review]

File: src/includes/set-release/xamarin.mdx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+```csharp {tabTitle:C#}
+using Sentry;
```
Comment (bruno-garcia) [inline src/includes/set-release/xamarin.mdx:2]: Isn't this `Sentry.Xamarin`?

Comment (lucas-zimerman) [reply src/includes/set-release/xamarin.mdx:2]: we are using namespace Sentry for the public classes.

**Pull request merged**